adapters/bwx/params_test.go

```diff
@@ -0,0 +1,53 @@
+package bwx
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+var validParams = []string{
+	`{"env":"boldwinx-stage", "pid":"123456"}`,
+}
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderBWX, json.RawMessage(validParam)); err != nil {
+			t.Errorf("Schema rejected boldwinx params: %s", validParam)
+		}
+	}
+}
+
+var invalidParams = []string{
+	``,
+	`null`,
+	`true`,
+	`5`,
+	`[]`,
+	`{}`,
+	`{"some": "param"}`,
+	`{"env":"boldwinx-stage"}`,
+	`{"pid":"1234"}`,
+	`{"othervalue":"Lorem ipsum"}`,
+	`{"env":"boldwinx-stage", pid:""}`,
+	`{"env":"", pid:"1234"}`,
+}
```

Comment:
Consider adding a test case where env and pid are provided, but are the wrong type such as an integer or boolean instead of a string.